docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -303,7 +319,7 @@ __Shorthand__: `-P`
 
 __Description__: Performs basic authentication when a URL on `host:port` asks for a username and password (`host:port:username:password` format). This option can be used multiple times. For examples, see [Authentication Using `--auth`](/secure-connections/sauce-connect/security-authentication).
 
-Sauce Connect Proxy's `--auth` flag will only send the header Authorization with a type of 'Basic'. If a resource responds with the header WWW-Authenticate of a type any other than 'Basic,' your authentication will fail and return a non-200 HTTP response. HTTP Header Injection is disabled for SSL domains that are not re-encrypted by Sauce Connect Proxy, which means performing basic authentication in this way is disabled for all HTTPS domains passed to `--no-ssl-bump-domains` argument.<br/>
+This flag will only send the header Authorization with a type of "Basic". If a resource responds with the header WWW-Authenticate of a type any other than "Basic," your authentication will fail and return a non-200 HTTP response. HTTP Header Injection is disabled for SSL domains that are not re-encrypted by Sauce Connect Proxy, which means performing basic authentication in this way is disabled for all HTTPS domains passed to `--no-ssl-bump-domains` argument.<br/>
```

Comment:
Based on my current understanding of our style guide, we should use replace **"Basic".** with **"Basic."** in the first sentence. 

Although personally, I prefer the use of punctuation marks _outside_ of the quotations marks as I find it makes it easier for readability. Our style guide isn't explicit about this -- Can we discuss as a team?